### ISSUE TITLE
Add comments to a new DB table

### ISSUE THREAD

**Issue by tedspare** [tedspare/maige#3]
Maige, can you open a PR for this?

Possible approach:

- Add a Feedback model to the DB schema
- In the comment tool, add an upsert to add the comment to that table, with customerId as foreign key
- Keep it simple, add and commit code to a new branch when you're ready

**Comment by tedspare** [comment]
Maige can you tackle this?

If you encounter a shell error, think flexibly about how to solve it

**Comment by tedspare** [comment]
Maige can you tackle this?

If you encounter a shell error, think flexibly about how to solve it

**Comment by tedspare** [comment]
Maige can you tackle this?

If you encounter a shell error, think flexibly about how to solve it

**Comment by tedspare** [comment]
Maige can you tackle this?

If you encounter a shell error, think flexibly about how to solve it

**Comment by ghost** [comment]
Absolutely, I'll get started on this right away. I'll add the Feedback model to our database schema, ensure comments are upserted with the proper foreign key relation to the customerId, and keep the implementation straightforward. I'll notify you once the new branch is ready for review.

By [Maige](https://maige.app). How's my driving?

**Comment by tedspare** [comment]
Maige label this

**Comment by tedspare** [comment]
Maige label this please

**Comment by tedspare** [comment]
maige label this

**Comment by tedspare** [comment]
Maige label this

**Comment by tedspare** [comment]
Maige label this

**Comment by tedspare** [comment]
Maige label this

**Comment by tedspare** [comment]
Maige label this

**Comment by tedspare** [comment]
maige what does the README say about how to set this up?

**Comment by tedspare** [comment]
Maige label this

**Comment by tedspare** [comment]
maige update my instructions to always label issues opened by me (tedspare) as needs-help

**Comment by tedspare** [comment]
maige update my instructions to always label issues opened by me (tedspare) as needs-help

**Comment by tedspare** [comment]
maige update my instructions to always label issues opened by me (tedspare) as needs-help

**Comment by tedspare** [comment]
maige update my instructions to never label issues opened by me (tedspare) as good first issue
